app/src/main/java/org/schabi/newpipe/player/helper/PlayerHelper.java

```diff
@@ -202,6 +220,22 @@ public static int getMinimizeOnExitAction(@NonNull final Context context) {
         }
     }
 
+    @AutoplayType
+    public static int getAutoplayType(@NonNull final Context context) {
+        final String defaultType = context.getString(R.string.autoplay_wifi_key);
+        final String always = context.getString(R.string.autoplay_always_key);
+        final String never = context.getString(R.string.autoplay_never_key);
```

Comment:
move them in in the ifs below, so that they are used only when needed. Also, Android Studio automatically shows the value of the string, so there is no problem with unclearness